app/controllers/manage_metadata_controller.rb

```diff
@@ -2,6 +2,31 @@
 class ManageMetadataController < ApplicationController
   include ManageMetadataHelper
```

Comment:
I know you didn't add this, but `app/helpers` are supposed to be view helpers. Since you are doing a little bit of refactoring, can you look at removing the need for including ManageMetadataHelper here